src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.max-age=0/accept/client.rpt

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newProxyAcceptRef ${nukleus:newRouteRef()} # external scope
+property date ${http_cache:date()} # shared property
+
+connect await ROUTED_PROXY
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
+                        ${http:header(":path", "/path")}
+                        ${http:header(":authority", "localhost:8080")}
+                        ${http:header("cache-control", "max-age=0")}
+
+
+connected
+
+write close
+read nukleus:begin.ext ${http:header(":status", "200")}
+                       ${http:header("content-length", "11")}
+                       ${http:header("cache-control", "max-age=0")}
```

Comment:
I believe this relates to this issue: https://github.com/reaktivity/nukleus-http-cache.spec/issues/16.  Which is specifically max-age 0 on [request cache directives](https://tools.ietf.org/html/rfc7234#section-5.2.1) NOT [response cache directives](https://tools.ietf.org/html/rfc7234#section-5.2.2).  

I believe response cache directive tests are already implemented but not named as such.  I'd suggest fixing the name in the already existing tests to reflect that.  AND  changing the name of this test to reflect that (maybe "request.cache.max-age=0").

Then remove the max-age=0 in the response part of this test.  Note, you will need something that identifies the response as cache-able though (one way to do this is max-age=15).  This is to ensure that the second request with max-age=0 is bypassing the cache because of max-age=0 NOT because there isn't anything in the cache :-)